docs/dev-readme.md

```diff
@@ -0,0 +1,52 @@
+# Developers Guide
+
+## Swagger UI
+
+### View REST API Specification from Swagger Editor UI
+
+Open https://editor.swagger.io/ in a browser. Click `File` - `Import file` from top menu and open the file [$WORKSPACE/redhat-cne/rest-api/v2/swagger.json](../v2/swagger.json). This loads the REST API specification like the following screenshot:
+
+![Alt text](swagger-editor.png "Swagger Editor")
+
+### Interact with REST-API in Swagger UI
+
+You can interact with API endpoint by click `Try it out`, enter required parameters and click `Execute`.
+This requires a REST-API server to be deployed at backend and accessible from localhost.
+
+#### Interact with cloud-event-proxy as the REST-API server
+
+When using cloud-event-proxy sidecar in linuxptp-daemon pod as the REST-API server, the API port in container needs to be port-forwarded to localhost.
+```
+oc port-forward -n openshift-ptp ds/linuxptp-daemon 9043:9043 &
+```
+
+In addition, API requests to a port-warded port could be rejected by the browser due to CORS. A chrome browser extension such as https://chromewebstore.google.com/detail/moesif-origincors-changer/digfbfaphojjndkpccljibejjbppifbc can be used to  allow cross-domain requests directly from browser without receiving Cross Origin Errors.
+
```

Comment:
Since this is third party browser extension , do mention of any security issues . for an example  remind to turn off this plugin when not used..

Reply:
added note of caution.